ui/src/main/java/edu/wpi/grip/ui/MainWindowController.java

```diff
@@ -59,6 +66,7 @@
   private Pane aboutPane;
   @FXML
   private StatusBar statusBar;
+  private Label elapsedTimeLabel; // TODO improve the layout of this label
```

Comment:
I would move the Pipeline status into its own label and then center both (i.e. do not use statusBar.setText() for the status)